diff --git a/thirdparty/pyibow/CMakeLists.txt b/thirdparty/pyibow/CMakeLists.txt
index 536c22d0..667262a1 100644
--- a/thirdparty/pyibow/CMakeLists.txt
+++ b/thirdparty/pyibow/CMakeLists.txt
@@ -41,7 +41,7 @@ if(BUILD_WITH_MARCH_NATIVE)
     endif()
 endif() 
 
-add_subdirectory(modules/pybind11)
+find_package(pybind11 REQUIRED)
 
 find_package(OpenCV REQUIRED)
 find_package(Boost REQUIRED COMPONENTS system filesystem serialization)
